tests/unit/test_redfish.py

```diff
@@ -555,11 +558,92 @@ def mock_get_response(uri):
         mock_redfish_obj.get.assert_any_call("/redfish/v1/Systems/s1/Storage/STOR1")
         mock_redfish_obj.get.assert_any_call("/redfish/v1/Systems/s1/Storage/STOR2")
 
+    @patch("prometheus_hardware_exporter.collectors.redfish.redfish_client")
+    @patch(
+        "prometheus_hardware_exporter.collectors.redfish.redfish_utilities.collections.get_collection_ids"  # noqa: E501
+    )
+    @patch(
+        "prometheus_hardware_exporter.collectors.redfish.redfish_utilities.systems.get_system_ids"
+    )
+    def test_12_non_standard_storage_uri_name(
```

Comment:
The logic in test_11 and test_12 are almost the same. Please think about how to reduce it.

Reply:
> I'm curious why we must use "numbers" to sort the test cases in code.
Whenever you add new test case in the middele , it impacts all the other test cases.

I'm onboard with removing these test numbers altogether. It definitely is a pain to re-order them while adding a new test case in the middle.

> The logic in test_11 and test_12 are almost the same. Please think about how to reduce it.

This is because the testing scenario for both is only slightly different -- storage name in URI is either `Storage` or `Storages`. I can put this as part of one test but then it won't be testing only one thing. Let me think about if there's a way to simplify these tests.